example_app_generator/generate_action_mailer_specs.rb

```diff
@@ -14,15 +14,20 @@
       end
     end
   CODE
-  gsub_file 'config/initializers/action_mailer.rb',
-            /ExampleApp/,
-            Rails.application.class.parent.to_s
+
+  rails_parent =
+    if Rails.version.to_f >= 6.0
+      Rails.application.class.module_parent.to_s
+    else
+      Rails.application.class.parent.to_s
+    end
```

Comment:
@benoittgt this is what replaced `parent` in 6.0 and above, I'm not sure if the 6.1 failure is real or a flake, I'll restart it.

Reply:
Nice finding!